### ISSUE TITLE
⌘-A does not select the entire page content on Google Chrome

### ISSUE THREAD

**Issue by Clue88** [upenn/irs#4]
Chrome 107.0.5304.87 running on MacOS 12.6.

How to Reproduce:
* Click "Class History"
* Select all using ⌘-A
* Paste the contents into the IRS tool
* Press "apply courses to requirements"

An "error parsing courses" error is returned. My guess is that copying all of the page contents works differently in different browsers—trying the same steps in Safari worked fine for me.

**Comment by devietti** [comment]
Very strange! Command-A works for me in Chrome. You should be able to use the `Edit` menu and then `Select All` (and that will show whatever keyboard shortcut is active for your setup).

**Comment by Clue88** [comment]
*link redacted*

Here's a diff between what's copied from Chrome (left) and from Safari (right)—I might try to take a look at exactly how the parser works later, but this might help diagnose the problem. It seems like the formats might be ever so slightly different?

**Comment by devietti** [comment]
I removed your link since it has your classes and grades listed. There is, however, definitely information missing in the Chrome version (some of the section headers, which are currently used to determine which worksheet to show), and I can repro the behavior you're seeing. I had only tested the Class History view in Safari before - thanks for reporting!

**Comment by nick-cirillo** [comment]
Can also reproduce on Firefox 106.0.1 on Windows 11 22H2 22621.674

**Comment by devietti** [comment]
Ok, I have added parsing of the subset of text that Chrome and Firefox (both on Mac) grab; they seem to work the same way. This makes it hard to parse the courses used in minors, unfortunately, but we can get the major and the course list. 

If someone can confirm it works better now I think we can close this.

**Comment by nick-cirillo** [comment]
Unfortunately, it looks like I can still reproduce the error on Firefox with Ctrl+A. Also, I'm not sure if the popup is asking us to select the entire page with Ctrl+A, or to _not_ select the entire page.

**Comment by Clue88** [comment]
I can confirm that the parsing now works on Chrome. Thanks!

**Comment by AaDalal** [comment]
Chrome and Firefox on Windows 11 also reproduce the issue. Firefox inside an Ubuntu VMware VM also reproduced (I think that's probably related to how VMware interfaces with the host OS).

**Comment by devietti** [comment]
It worked ok for me even on c75c49ef9 with Chrome 107.0 on Windows 10. However, I did encounter this problem with Firefox 102.5 on Windows 10, which has been fixed (at least in my testing) as of aa0e210. 

I am going to optimistically close this issue. Please re-open if you are still encountering this problem!
